setup.Rmd

```diff
@@ -109,45 +93,6 @@ devtools::install_github("r-lib/devtools")
 devtools::install_github("r-lib/usethis")
 ```
 
-## R build toolchain {#setup-tools}
```

Comment:
I'm not sure we can entirely remove this. But in the name of merging this PR, which is overwhelmingly helpful, I can accept the deletion. But will you open an issue about re-inserting some suitable stub re: setting your machine for package development? And paste in the text that we are deleting?

Reply:
Will do - sorry I was a bit overzealous!

Reply:
Opened in #937